addons/l10n_es_edi_facturae/models/account_move.py

```diff
@@ -192,7 +192,7 @@ def _l10n_es_edi_facturae_inv_lines_to_items(self, conversion_rate=None):
                 continue
             invoice_line_values = {}
 
-            price_before_discount = line.currency_id.round(line.price_subtotal / (1 - line.discount / 100.0))
+            price_before_discount = line.currency_id.round(line.price_subtotal / ((1 - line.discount / 100.0) or 1))
```

Comment:
Hi !
That means price_before_discount is now 0 instead of the real price before discount. Not a problem ?

Reply:
@jbw-odoo Changes are done!

Reply:
Ok, lgtm, can you please add a test ?

Reply:
@jbw-odoo Changes are done!